integrations/datadog/.port/resources/port-app-config.yaml

```diff
@@ -57,3 +57,21 @@ resources:
           relations:
             monitors: .monitor_ids | map(tostring)
             services: '.monitor_tags + .tags | map(select(startswith("service:"))) | unique | map(split(":")[1])'
+  - kind: sloHistory
+    selector:
+      query: 'true'
+      fromTimestamp: 1711411200 # The from timestamp for the query window in epoch seconds.
+      toTimestamp: 1711898238
```

Comment:
how is this expected to be used in the mapping structure? will this need to be updated every day? I think we need a more useable solution? please elaborate